database/mssql/scripts/dbscripts/PSP_PIMS_S24_00/Test Data/05_DML_LEASE_PROPERTY.sql

```diff
@@ -0,0 +1,1317 @@
+INSERT INTO [dbo].[PIMS_PROPERTY]([PID], [PIN], [PROPERTY_TYPE_CODE], [SURPLUS_DECLARATION_TYPE_CODE], [PROPERTY_STATUS_TYPE_CODE], [PROPERTY_DATA_SOURCE_TYPE_CODE], [PROPERTY_DATA_SOURCE_EFFECTIVE_DATE], [PROPERTY_CLASSIFICATION_TYPE_CODE], [PROPERTY_TENURE_TYPE_CODE], [NAME], [DESCRIPTION], [ADDRESS_ID], [REGION_CODE], [DISTRICT_CODE], [PROPERTY_AREA_UNIT_TYPE_CODE], [LAND_AREA], [LAND_LEGAL_DESCRIPTION], [ENCUMBRANCE_REASON], [IS_SENSITIVE], [IS_OWNED], [IS_PROPERTY_OF_INTEREST], [IS_VISIBLE_TO_OTHER_AGENCIES], [ZONING], [ZONING_POTENTIAL], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY])
```

Comment:
@dfilteau this appears to be based on an old version of the test data. Can you pull, and replace this with the contents of the file from PSP_PIMS_S23_00 please?

Reply:
The Test Data folder is a legacy folder that I intended to delete.  Do you still need it?

Reply:
Yes. This is how we build our dev databases. Feel free to not include it in your PR's, we often need to update it at the start of sprints.